File: covid-19-backend/CovidHelp/angular/src/app/generated-api/models/config-app-view-model.ts
/* tslint:disable */
import { ModelTypes } from './model-types';
import { ModelFieldDefinitionViewModel } from './model-field-definition-view-model';
export interface ConfigAppViewModel {
  version?: string;
  allModelTypes?: Array<ModelTypes>;
  models?: Array<ModelFieldDefinitionViewModel>;
}

File: covid-19-backend/CovidHelp/Models/ViewModels/ConfigAppViewModel.cs
using NetCoreCommons.DynamicForms.Models;
using CovidHelp.Models.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CovidHelp.Models.ViewModels
{
    public class ConfigAppViewModel
    {
        public string Version { get; set; }
        public IEnumerable<ModelTypes> AllModelTypes { get; set; }
        public IEnumerable<ModelFieldDefinitionViewModel> Models { get; set; }
    }
}
